javascript/javascript1/week4/assistantvoice.js

```diff
@@ -0,0 +1,33 @@
+function getReply(order) {
```

Comment:
You are not getting the name when the `order` is `Hello my name is ???`
You can do it like this
```js
function getReply(order) {
  let name;
  const commandWords = order.split(' ');

  if (order.includes('hello my name is')) {
    name = commandWords[commandWords.length - 1];
    return `Nice to meet you ${name}`;
  }
```
or like this
```js
function getReply(order) {
  let name;
  if (order.includes('hello my name is')) {
    name = order.replace('hello my name is', '');
    return `Nice to meet you ${name}`;
  }
```
That means whatever is the name you will get it. Not only if it is Benjamin.